src/main/java/gc/cafe/entity/OrderItems.java

```diff
@@ -0,0 +1,76 @@
+package gc.cafe.entity;
+
+import gc.cafe.controller.dto.common.entitydto.requestdto.OrderItemsRequestDTO;
+import gc.cafe.controller.dto.common.entitydto.responsedto.OrderItemsResponseDTO;
+import gc.cafe.controller.dto.common.entitydto.responsedto.ProductsResponseDTO;
+import jakarta.persistence.*;
+import lombok.AccessLevel;
+import lombok.Builder;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+
+import java.time.LocalDateTime;
+
+@Entity
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@Table(indexes = @Index(name = "idx_order_id", columnList = "order_id"))
+public class OrderItems {
+    @Id
+    @GeneratedValue
+    private Long seq;
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "orderId")
+    private Orders order;
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "productId")
+    private Products product;
+    @Column(nullable = false, length = 50)
+    private String category;
+    @Column(nullable = false)
+    private Long price;
+    @Column(nullable = false)
+    private int quantity;
+    @CreationTimestamp
+    private LocalDateTime createdAt;
+    @UpdateTimestamp
+    private LocalDateTime updatedAt;
```

Comment:
사소하지만 각 줄간에 여백이 있으면 좋을 것 같습니다! 취향 차이긴합니다.
```suggestion
    @Id
    @GeneratedValue
    private Long seq;
    
    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "orderId")
    private Orders order;
    
    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "productId")
    private Products product;
    
    @Column(nullable = false, length = 50)
    private String category;
    
    @Column(nullable = false)
    private Long price;
    
    @Column(nullable = false)
    private int quantity;
    
    @CreationTimestamp
    private LocalDateTime createdAt;
    
    @UpdateTimestamp
    private LocalDateTime updatedAt;
```

Reply:
헉 그렇네요